spipe/Makefile.BSD

```diff
@@ -10,6 +10,9 @@ LDADD	=	-lrt
 LDADD_REQ	=	-lcrypto
 LDADD_REQ	+=	-lpthread
 
+# Useful relative directories
+LIBCPERCIVA_DIR	=	../libcperciva
```

Comment:
Wasn't the idea that this was going to make use of SUBDIR_DEPTH?

Reply:
Unfortunately, `.include <bsd.prog.mk>` comes at the end, so SUBDIR_DEPTH isn't defined when make parses this line.  Also, it wants to find `SRCS += foo.c` before it does the late parsing of include files.

I'm not saying it's **impossible** to get it to work, but after 3 or 4 hours, it was looking like it would involve a lot of black magick and I'd need to dig into the source file(s) of BSD make.  And since it could be achieved with a single manually-written line in each `Makefile.BSD`, I figured that was the best option.

(Note that `SUBDIR_DEPTH` in each subdir (via `Makefile.inc`) is still useful, as that allows the top-level `Makefile.BSD` to include the correct directory to find the top-level `cpusupport-config.h`.)